content/source/docs/cloud/sentinel/manage-policies.html.md

```diff
@@ -20,7 +20,11 @@ Sentinel Policies are rules which are enforced on Terraform runs to validate tha
 [users]: ../users-teams-organizations/users.html
 [workspaces]: ../workspaces/index.html
 
-**Policies** consist of a Sentinel policy file and an enforcement level.
+**Policies** are written using the Sentinel [language](https://docs.hashicorp.com/sentinel/concepts/language). Policies are the guardrails that prevent Terraform runs from performing dangerous actions. Upon evaluation, policies will adhere to a predefined [enforcement level](#enforcement-levels). Policies are either managed individually or as part of a versioned policy set. 
+
+Individual policies allow the policy to be edited directly in the Terraform UI, whereas versioned policy sets are defined as individual policy files and are stored in a supported VCS provider or uploaded via the Terraform API.
+
+-> **Note:** Individually managed policies is a deprecated feature in Terraform Cloud and recommend versioned policy sets as the primary method of enforcing policies. 
```

Comment:
How about making this recommendation stronger, and mentioning the correct interface before the incorrect interface? 

```suggestion
**Policies** are written using the Sentinel [language](https://docs.hashicorp.com/sentinel/concepts/language). Policies are the guardrails that prevent Terraform runs from performing dangerous actions. Upon evaluation, policies will adhere to a predefined [enforcement level](#enforcement-levels). 

Policies are managed as parts of versioned policy sets, which allow individual policy files to be stored in a supported VCS provider or uploaded via the Terraform Cloud API.

-> **Note:** It's also possible to manage policies and policy sets individually. However, this is a deprecated feature in Terraform Cloud, and we recommend always using versioned policy sets to manage policies. 
```

Reply:
Thank @nfagerlund this is a much better approach